hack/install-deps.sh

```diff
@@ -125,3 +126,7 @@ checkout_repo ${CONTAINERD_PKG} ${CONTAINERD_VERSION}
 cd ${GOPATH}/src/${CONTAINERD_PKG}
 make
 ${sudo} make install -e DESTDIR=${CONTAINERD_DIR}
+
+#Install crictl
+go get -u ${CRICTL_PKG}
```

Comment:
You may want to make sure we are using the correct version of CRICTL.
Probably using `checkout_repo`.

Reply:
I thought about it. Wasnt sure if this is strictly necessary.

Reply:
CRI is not stabilized yet, it may change in the future. If we don't enforce the version here, user may not be able to use this script in older branch, because CRICTL has been updated to newer CRI.